src/components/Admin/AdminComponents/AlertDialog.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+const AlertDialog = props => {
```

Comment:
You can use withStyles to customize css for material UI components:
https://material-ui.com/customization/components/